controllers/quoteattestation_controller.go

```diff
@@ -154,91 +155,61 @@ func (r *QuoteAttestationReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		// object being deleted, just ignore
 		return ctrl.Result{}, nil
 	}
+	// Do not handle quote attestation request, that supposed to be
+	// handled by the signer (CSR, CR) controllers.
+	if attestReq.Spec.Type == v1alpha1.RequestTypeQuoteAttestation {
```

Comment:
But that code's not part of this PR, right?

Reply:
Yes, that will be a future change. The reconciler in `quoteattestation_controller.go` is only key provision requests.